File: server-application-server/source/java/ch/systemsx/cisd/openbis/public/resources/api/v3/as/dto/session/id/SessionInformationPermId.js
define([ "stjs", "as/dto/common/id/ObjectPermId", "as/dto/session/id/ISessionInformationId" ], function(stjs, ObjectPermId, ISessionInformationId) {
	var SessionInformationPermId = function(permId) {
		ObjectPermId.call(this, permId);
	};
	stjs.extend(SessionInformationPermId, ObjectPermId, [ ObjectPermId, ISessionInformationId ], function(constructor, prototype) {
		prototype['@type'] = 'as.dto.session.id.SessionInformationPermId';
		constructor.serialVersionUID = 1;
	}, {});
	return SessionInformationPermId;
})

File: api-openbis-java/source/java/ch/ethz/sis/openbis/generic/asapi/v3/dto/session/id/SessionInformationPermId.java
package ch.ethz.sis.openbis.generic.asapi.v3.dto.session.id;

import ch.ethz.sis.openbis.generic.asapi.v3.dto.common.id.ObjectPermId;
import ch.systemsx.cisd.base.annotation.JsonObject;

@JsonObject("as.dto.session.id.SessionInformationPermId")
public class SessionInformationPermId extends ObjectPermId implements ISessionInformationId
{

    private static final long serialVersionUID = 1L;

    public SessionInformationPermId(String permId)
    {
        super(permId);
    }

    //
    // JSON-RPC
    //

    @SuppressWarnings("unused")
    private SessionInformationPermId()
    {
        super();
    }

}
